code/game/machinery/computer/card.dm

```diff
@@ -528,6 +571,44 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 				to_chat(usr, "<span class='notice'>[j.title] has been successfully [priority ?  "prioritized" : "unprioritized"]. Potential employees will notice your request.</span>")
 				playsound(src, 'sound/machines/terminal_prompt_confirm.ogg', 50, 0)
 
+		if ("adjust_pay")

```

Comment:
You should add checks here to make sure the user has the ability to adjust pay and isn't just exploiting hrefs

Reply:
So I'd have to recheck the departments that the user can adjust the pay of too?